### PR TITLE
Added cart wishlist component

### PR DESCRIPTION
**Pull request by Mohitk391**

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <link rel="stylesheet" href="https://mhp-easy-ui.netlify.app/components/component-styles.css">
+    <link rel="stylesheet" href="/cart.css">
+</head>
+<body>
+    <nav class="nav-bar">
+        <a class="nav-bar-logo" href="/index.html">FURN-EASY</a>
+        <input type="nav-bar-search" placeholder="Search"/>
+        <div class="nav-bar-buttons">
+            <button class="login">Logout</button>
+            <a href="/wishlist.html" class="nav-bar-icon"><i class="fa fa-heart-o" aria-hidden="true"></i></a>
```
Comment (parvez-rashid) [inline cart.html:20]: Great that you've `aria-` properties for good accessibility.

File: wishlist.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}
+.wishlist-title{
+    text-align: center;
+    font-size: 1.5rem;
+    font-weight: 600;
+    padding: 1rem;
+}
+
+.main-body{
+    display: flex;
```
Comment (parvez-rashid) [inline wishlist.css:17]: You could've created utility class just for `display: flex`, would've made it easier to use everywhere. e.g., 
```.disp-flex {display: flex;}```

File: wishlist.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                </div>
+            </div>
+            <div class="card">
+                <div class="image">
+                    <img src="/assets/sofa-1.jpeg" alt="card-image">
+                </div>
+                <div class="card-content">
+                    <div class="card-title">Sofa</div>
+                    <div class="card-sub-title">Rs. 4000</div>
+                    <div class="footer flex flex-center">
```
Comment (parvez-rashid) [inline wishlist.html:81]: I see that there is in fact a utility class used here for setting display to flex. Try to use this in other components above as well.

File: cart.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    border: 1px solid var(--clr-black);
+}
+
+.card-title{
+    text-align: center;
+}
+.card-sub-title{
+    text-align: center;
+    
+}
```
Comment (parvez-rashid) [inline cart.css:31]: Suggestion: To make code cleaner, do it like this:

```
.card-title,
.card-sub-title {
    text-align: center;
}
```

Better yet, use utility class for `text-align: center` and apply it wherever needed.

**Review by parvez-rashid — APPROVED** [review]
Please include live preview or screenshots of the UI. Code-wise I have a few comments below, have a look. Other than that, no issues with the code, you can merge it.

**Review by nikhiljugale007 — COMMENTED** [review]
Other than changes suggested by parvez-rashid everything is good to merge. 
Suggestion: You add preview link in pr description

File: cart.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    align-items: center;
+    gap:0.5rem;
+}
+.card-button-cart{
+    font-size: 1.2rem;
+    width: 100%;
+    border:none;
+    color: var(--clr-white);
+    border-radius: 1rem;
+    background-color: var(--cart-blue);
```
Comment (anasm11) [inline cart.css:58]: color variable names can be more generalised, would be easy in case you change the color later.

**Review by anasm11 — APPROVED** [review]
The code looks good to merge.

**Pull request merged**